tests/unit/src/unit/hxcpp_issues/Issue9194.hx

```diff
@@ -0,0 +1,24 @@
+package unit.hxcpp_issues;
+
+
+class Issue9194 extends Test {
+
+	@:analyzer(no_optimize)
+	function test() {
+		#if (cpp && !cppia)
```

Comment:
Why the test is disabled for cppia?

Reply:
Mainly because I've never tried cppia :P, let me give it a try